back/identify/Github/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+# Create your models here.
+
+class Github(models.Model):
+    account_id = models.BigIntegerField(_("Account ID"), primary_key=True)
+    node_id = models.CharField(_("Node ID"), max_length=50)
+    name = models.CharField(_("Name"), max_length=40)
+    twitter_username = models.CharField(_("Twitter Username"), max_length=50, blank=True, null=True)
+    location = models.CharField(_("Location"), max_length=200)
+    hireable = models.BooleanField(_("Hireable"))
+    email = models.EmailField(_("Email"), max_length=254)
+    avatar_url = models.URLField(_("Avatar"), max_length=200)
```

Comment:
Make sure that every user on Github has an avatar URL
if they didn't, add null and blank True to avatar_url field